src/posthog-core.js

```diff
@@ -557,6 +557,7 @@ PostHogLib.prototype.push = function (item) {
  * @param {Object} [properties] A set of properties to include with the event you're sending. These describe the user who did the event or details about the event itself.
  * @param {Object} [options] Optional configuration for this capture request.
  * @param {String} [options.transport] Transport method for network request ('XHR' or 'sendBeacon').
+ * @param {Date} [options.timestamp] Timestamp in ISO 8601 format.  If not set, it'll automatically be set to the current time.
```

Comment:
I think this should be one space before "If not set".
BTW, this is not ISO 8601, but a `Date object` – let's make the comment accurate.